geoalchemy2/elements.py

```diff
@@ -136,6 +137,27 @@ class WKBElement(_SpatialElement):
     geom_from = 'ST_GeomFromWKB'
     geom_from_extended_version = 'ST_GeomFromEWKB'
 
+    WKB_XDR = 0  # Most Significant Byte First
```

Comment:
not used?

Reply:
yep, should I move it in docstring? (to keep the postgis enum as documentation purpose)

Reply:
moved to docstring